service/src/main/java/org/apache/celeborn/server/common/service/store/db/DbServiceManagerImpl.java

```diff
@@ -61,6 +61,7 @@ public DbServiceManagerImpl(CelebornConf celebornConf, ConfigService configServe
     this.clusterId = createCluster(getClusterInfoFromEnv());
   }
 
+  @SuppressWarnings("JavaUtilDate")
```

Comment:
Curious, why this is suppressed here? Is this planned to be addressed in a follow-up PR?

Reply:
@venkata91, I referred to [JavaUtilDate](https://errorprone.info/bugpattern/JavaUtilDate) but this could not lead to bugs. Do you prefer the `java.time` APIs?

Reply:
Yes, migration to `java.time` APIs.

Reply:
@mridulm, WDYT?

Reply:
Wouldn't it be better to move to `java.time` APIs to avoid any future evolution issues? Do you see any issues with migrating it now? Basically any backwards compatibility issues?

Reply:
`Instant` and `LocalDate` is not directly handled by apache ibatis, is it ? I am not very confident - but given this is used in that context, we have to ensure it is supported (else we will end up writing type handler, etc ... I dont think it is worth that).

Reply:
@mridulm did you mean ratis or ibatis (looks like this is moved ro attic :))?

Reply:
@mridulm, this change makes sense to you, right?

Reply:
@mridulm Looks like you meant `ibatis` only - seeing its usage in Celeborn.

Reply:
> Instant and LocalDate is not directly handled by apache ibatis, is it ? I am not very confident - but given this is used in that context, we have to ensure it is supported (else we will end up writing type handler, etc ... I dont think it is worth that).

I see the concern here. I'm fine with not handling this now given Apache

Reply:
@mridulm @SteNicholas Having said that, looks like the [type-handlers are merged in to mybatis core since 3.4.5](https://stackoverflow.com/questions/25113579/java-8-localdate-mapping-with-mybatis) [corresponding [PR](https://github.com/mybatis/mybatis-3/pull/974)] and Celeborn is already on 3.5.x - this looks to be fine, no?

Reply:
@venkata91, changing to `Instant` type of [type-handlers are merged in to mybatis core since 3.4.5](https://stackoverflow.com/questions/25113579/java-8-localdate-mapping-with-mybatis) support has type compatibility problem for the database implementation of dynamic config. Right?

Reply:
Can you elaborate a bit? I don't think I i got it.

Reply:
@venkata91, when `gmtCreate` and `gmtModify` are updated to `Instant` type. Does the `gmt_create` and `gmt_modify` in `celeborn-0.5.0-mysql.sql` need to change?

Reply:
Looks like `java.time.Instant` is mapped to TIMESTAMP based on the [InstantTypeHandlerTest](https://github.com/harawata/mybatis-3/blob/2da2275185e8bb185d51eb522120ac32e5635225/src/test/java/org/apache/ibatis/type/usesjava8/InstantTypeHandlerTest.java). Looks like it should be handled, no?

Reply:
@venkata91, I have addressed comments in https://github.com/apache/celeborn/pull/2555/commits/9e0ff9fc000074ee6a78d0b3a91343125a89910a. `ConfigServiceSuiteJ` could verify this change. PTAL. cc @RexXiong.

Reply:
Looks good. Thanks!